Node/src/TeamsMessage.ts

```diff
@@ -41,19 +41,54 @@ export enum MentionTextLocation {
   AppendText
 }
 
+export class MentionEntity {
```

Comment:
Unit tests?
By the way the indentation in https://github.com/OfficeDev/BotBuilder-MicrosoftTeams/blob/master/Node/test/TeamsMessage.js is all over the place. (Mix of tabs and spaces?) Please fix.

Reply:
@aosolis Hi Adrian we don't have any unit test for now. I can set all them up later.